### ISSUE TITLE
Currently missing many planets

### ISSUE THREAD

**Issue by bhorowitz** [modichirag/Planets#2]
Currently we have multiple missing planets, yet two non-planets (mummy and pluto). This should be corrected in next stable release.

**Comment by modichirag** [comment]
I agree. I am awaiting results from the recent Planet IX workshop before updating things for next stable release
